Modules/Filtering/BinaryMathematicalMorphology/test/itkBinaryClosingByReconstructionImageFilterTest.cxx

```diff
@@ -28,42 +28,61 @@ int
 itkBinaryClosingByReconstructionImageFilterTest(int argc, char * argv[])
 {
 
-  if (argc != 6)
+  if (argc < 6)
```

Comment:
If argc == 99 this will not be run, even though the command line is wrong.

Reply:
Changed in 8e4315a.